WalletWasabi.Fluent/ViewModels/AddWallet/WelcomePageViewModel.cs

```diff
@@ -24,13 +22,17 @@ public WelcomePageViewModel(AddWalletPageViewModel addWalletPage)
 
 		SelectedIndex = 0;
 		NextCommand = ReactiveCommand.Create(OnNext);
+		var canGoBackChanged = this.WhenAnyValue(x => x.SelectedIndex, i => i > 0);
+		BackCommand = ReactiveCommand.Create(() => SelectedIndex--, canGoBackChanged);
+		canGoBackChanged
+			.Subscribe(canGoBack => EnableBack = canGoBack);
```

Comment:
This could be replaced with a binding to an observable.

Reply:
I completely forgot... Thanks!